src/main/interface/cli.c

```diff
@@ -859,28 +859,33 @@ static void printAux(uint8_t dumpMask, const modeActivationCondition_t *modeActi
                 && mac->auxChannelIndex == macDefault->auxChannelIndex
                 && mac->range.startStep == macDefault->range.startStep
                 && mac->range.endStep == macDefault->range.endStep
-                && mac->modeLogic == macDefault->modeLogic;
+                && mac->modeLogic == macDefault->modeLogic
+                && mac->linkedTo == macDefault->linkedTo;
             const box_t *box = findBoxByBoxId(macDefault->modeId);
-            if (box) {
+            const box_t *linkedTo = findBoxByBoxId(macDefault->linkedTo);
+            if (box && linkedTo) {
```

Comment:
This is wrong. If `linkedTo` is 0 here we still have to print the line (for non-linked modes).

It's probably easiest and most extensible if we print `0` instead of `linkedTo->permanentId` in this case, and accept `0` as a second form of 'not linked' for setting the mode below.

Reply:
Yeah, you're right. Fixed. 0 works, too.